package.json

```diff
@@ -42,6 +42,7 @@
     "ember-load-initializers": "^0.6.0",
     "ember-osf": "git+https://github.com/CenterForOpenScience/ember-osf.git#4247d6d3e3e41aa29c3c8ee2c0cbe73fa54a7090",
     "ember-power-select": "^1.8.5",
+    "ember-quill": "0.0.14",
```

Comment:
It looks like you didn't remove ember-froala-editor. Please remove it so we don't install it again.